assets/css/includes/scroll-bar.css

```diff
@@ -3,17 +3,13 @@
     background: 0 0;
 }
 
-.list:not(.dark)::-webkit-scrollbar-track {
-    background: var(--code-bg);
-}
-
```

Comment:
I don't quite follow the changes on scroll bars here, but I don't notice any difference in my testing so I assume these are valid changes.

Reply:
yes this was redundant it seems. or some reason I don't remember about, but it doesn't affect it now.